### ISSUE TITLE
Wakatime language identification error

### ISSUE THREAD

**Issue by chenboshuo** [wakatime/wakatime#225]
My latex class file(`*.cls`) was identified  `OpenEdge ABL`
```text
OpenEdge ABL
~/Documents/MathModels/cumcm2/cumcm_template/thesis/cumcmthesis.cls

~/Documents/latex_template/cumcm_template/thesis/cumcmthesis.cls

```
![image](https://user-images.githubusercontent.com/43953268/89246394-9d61c180-d63d-11ea-8e34-d92631eb29d9.png)

-   [my cls file](https://github.com/chenboshuo/cumcm_template/blob/master/thesis/cumcmthesis.cls)

**Comment by egordorichev** [comment]
Didn't want to open a new issue for this, but yeah, it would be really neat, if you could actually specify/modify the language somehow. 

![image](https://user-images.githubusercontent.com/7851390/89410973-4f2beb80-d72d-11ea-9023-3995e78d5a0d.png)

Say I'm working with a little-known language, and it would be really neat if I could go into settings and define something like:

`*.myfiletype == My Awesome Language`

**Comment by alanhamlett** [comment]
> it would be really neat, if you could actually specify/modify the language somehow.

Yep, you can do that with a "Custom Rule" matching the end of the file:
https://wakatime.com/settings/rules

**Comment by alanhamlett** [comment]
> My latex class file(*.cls) was identified OpenEdge ABL

We'll fix this in a future wakatime-cli release, but for now you should use [custom rules](https://wakatime.com/settings/rules) to fix the language. Let's leave this issue open until the code fix is released.

**Comment by egordorichev** [comment]
Oh, my bad, huge thanks for pointing this feature out to me!
